.github/scripts/shellCheck.sh

```diff
@@ -1,48 +1,49 @@
 #!/bin/bash
 
-CURRENT_DIR=$(pwd)
-ROOT_DIR="$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")"
-
-cd "$ROOT_DIR" || exit 1
-
-source ./.github/scripts/shellUtils.sh
-
-declare -r DIRECTORIES_TO_IGNORE=(
-  './node_modules'
-  './vendor'
-  './ios/Pods'
-  './.husky'
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+readonly SCRIPT_DIR
+ROOT_DIR="$(dirname "$(dirname "$SCRIPT_DIR")")"
+readonly ROOT_DIR
+
+source "$SCRIPT_DIR"/shellUtils.sh
```

Comment:
Making relative to ROOT_DIR per comment above. Also while having the quotes in the middle of the string like that is technically valid, it's much more common to have it surrounding the whole string if the syntax allows. We should really add that to the style guide.

```suggestion
source "$ROOT_DIR/.github/scripts/shellUtils.sh"
```